net.sourceforge.GrandOrgue.yml

```diff
@@ -0,0 +1,69 @@
+app-id: net.sourceforge.GrandOrgue
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '20.08'
+command: GrandOrgue
+rename-desktop-file: GrandOrgue.desktop
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --filesystem=home
+  - --socket=system-bus
+  - --filesystem=xdg-run/pipewire-0
+cleanup:
```

Comment:
Since you use jack, the global cleanup phase should have 
```
- /lib/libjack*
- /lib/jack
```

This to completely remove jack and have the app use pw-jack that is in the SDK.

Without that it likely won't work.

Reply:
By likely I mean it doesn't work.

(I should actually now remove them from the shared-modules)

Reply:
if you update the shared-module submodule, you will no longer need to do the above.

Reply:
Updated the submodule in the next commit. Thanks!